torq.sh

```diff
@@ -221,8 +221,14 @@ usage() {
   exit 1
```

Comment:
Why do you need to change torq.sh in this PR?

Reply:
monit executes torq.sh from the root directory. Torq.sh uses `. setenv.sh` and obviously the `setenv.sh` is not in the root directory